examples/04-Icepak/Sherlock_Example.py

```diff
@@ -181,6 +194,30 @@
 
 ipk.assign_priority_on_intersections()
 
+###############################################################################
+# Compute power Budget
+# ~~~~~~~~~~~~~~~~~~~~~~~
+
+power_budget, total = ipk.post.power_budget("W" )
+print(total)
+
+###############################################################################
+# Compute power Budget
+# ~~~~~~~~~~~~~~~~~~~~~~~
+
+ipk.analyze_nominal()
+
+###############################################################################
+# Get Solution Data and Plot
+# ~~~~~~~~~~~~~~~~~~~~~~~
+
+face_u10 = [i.id for i in ipk.modeler["COMP_U10"].faces]
+plot1 = ipk.post.create_fieldplot_surface(face_u10, "SurfTemperature")
+plot1.export_image(os.path.join(ipk.working_directory, "temperature_u10.jpg"))
+face_u9 = [i.id for i in ipk.modeler["COMP_U9"].faces]
+plot2 = ipk.post.create_fieldplot_surface(face_u9, "SurfPressure")
+
```

Comment:
Maybe we could export the image the same way we export the image for the temperature around `u10`.